ethcore/evm/src/interpreter/shared_cache.rs

```diff
@@ -91,6 +109,27 @@ impl SharedCache {
 		jump_dests.shrink_to_fit();
 		Arc::new(jump_dests)
 	}
+
+	fn find_sub_destinations(code: &[u8]) -> Arc<BitSet> {
```

Comment:
this function is used together with `find_jump_destinations`, so I guess it could be merge into one avoiding extra pass

Reply:
Yeah you're right!